mito-ai/src/components/SelectedContextContainer.tsx

```diff
@@ -29,21 +40,46 @@ const SelectedContextContainer: React.FC<SelectedContextContainerProps> = ({ tit
         case 'db':
             icon = <DatabaseIcon />;
             break;
+        case 'active_cell':
+            icon = <CodeIcon />;
+            break;
     }
 
+    const handleClick = (): void => {
+        if (type === 'active_cell' && notebookTracker && activeCellID) {
+            // Find the cell 
+            const cell = getCellByID(notebookTracker, activeCellID);
+            if (cell) {
+                // Scroll to the cell
+                void notebookTracker.currentWidget?.content.scrollToCell(cell, 'center');
+                // Highlight the cell
+                setTimeout(() => {
+                    highlightCodeCell(notebookTracker, activeCellID);
+                }, 500);
+            }
+        } else if (onClick) {
```

Comment:
If for some reason the type was active cell, but the notebookTracker was not yet defined then we would try calling this. Should we just make the above check first checking if it is the active_cell and then move the other checks inside of that case to avoid that potentially incorrect code flow?